src/query.c

```diff
@@ -649,7 +649,9 @@ static IndexIterator *Query_EvalVectorNode(QueryEvalCtx *q, QueryNode *qn) {
       return NULL;
     }
   }
-  IndexIterator *it = NewVectorIterator(q->sctx, qn->vn.vq, child_it, q->status);
+  // Check if we can ignore the document score
+  bool ignoreScores = q->opts->flags & Search_IgnoreScores;
+  IndexIterator *it = NewVectorIterator(q->sctx, qn->vn.vq, child_it, ignoreScores, q->status);
```

Comment:
can we send `q` instead of extracting the `sctx`, `status`, `ignoreScores`?
`ignoreScores` is used to create the hybrid iterator only, so we can extract everything form `q`